bemani/backend/popn/unilab.py

```diff
@@ -391,9 +401,27 @@ def get_common_config(self) -> Tuple[Dict[int, int], bool]:
                 # Super Unilab BOOST! (0-2)
                 # Boost should be 120, 150, or 200, bemaniwiki has the explanation and it's based on the unlocks left to do
                 6: super_unilab_boost,
-                # Unknown event (0-6)
-                7: 6,
-                # Unknown event (0-2)
+                # Shutchou! pop'n quest Lively II ~nostalgia~ (0-6)
+                # When active, the following songs are available for unlock
+                # 1 - 1990 - Petit Queen
+                #     1991 - 波と凪の挟間で
+                #     1989 - Ketter
+                # 2 - 1984 - コルドバの女
+                #     1985 - say...but in vain
+                #     1992 - Northern Cross
+                # 3 - 1982 - Surf on the Light
+                #     1988 - Danza Pantera
+                #     1983 - バッドエンド・シンドローム
+                # 4 - 1993 - Un Happy Heart
+                #     1987 - 水晶塔のオルカ
+                #     1986 - virkatoの主題によるperson09風超絶技巧変奏曲
+                # 5 - 2017 - virkatoの主題によるperson09風超絶技巧変奏曲 upper
+                # 6 - Event Ended
+                7: popn_quest_lively_2,
```

Comment:
If you copy/paste these into a different program please make sure you update these to the corrected values.